webclient/impls/core/src/test/config/javascript/karma.ci.conf.js

```diff
@@ -22,6 +22,7 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
+      'src/test/config/javascript/context.js',
```

Comment:
Must be loaded before webjar dependencies, so that RequireJS is removed and these register globally.